### ISSUE TITLE
Explosions don't break backpacks

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#21039]
Probably should break it and spill everything out.

**Comment by iacore** [comment]
does backpack have hp?

**Comment by CatMagic1** [comment]
Anything that makes terrorism more realistic is a yes from me

**Comment by Endecc** [comment]
Makes sense if explosions come from within the backpack itself, but backpacks being given HP and being caught in any explosion at all sounds like a recipe for a super annoying mess. I'm in favor if it's kept narrow, being explosions coming from inside the backpack.

As another thought, contents inside the backpack could be catapulted around randomly. Create your own shrapnel grenade by chucking a frag in a backpack with 20 glass shards!

**Comment by iacore** [comment]
can you put a grenade in a bag?

**Comment by forgotmyotheraccount** [comment]
#21020 seems like it would address this, right? I feel like outright destroying backpacks would be overkill for players who have already had the misfortune of being bombed, whereas igniting them would at the very least give them a small window to extinguish it.

Of course, if an explosive contained inside a backpack detonates, then the backpack should definitely be destroyed.

**Comment by Equivocateur** [comment]
> can you put a grenade in a bag?

https://github.com/space-wizards/space-station-14/assets/109166122/76a3ec60-1a59-41e5-84bb-8b88b1f997c8

**Comment by iacore** [comment]
> > can you put a grenade in a bag?
> 
> sus.mp4

did the grenade explode **in** the bag?

**Comment by iacore** [comment]
> As you can visually see the grenades were in the bag

none of them disappeared from the bag, although that might be a visual bug.
